src/web/app/components/sessions-recycle-bin-table/sessions-recycle-bin-table.component.html

```diff
@@ -0,0 +1,71 @@
+<div class="row card-body recycle-bin-body" *ngIf="recycleBinFeedbackSessionRowModels.length !== 0">
+  <div class="col-12">
+    <h2 class="text-muted">
+      <i class="fas fa-trash-alt"></i> Deleted Feedback Sessions
+    </h2>
+    <div class="card bg-light top-padded">
+      <div class="card-header bg-secondary recycle-bin-header" (click)="isRecycleBinExpanded = !isRecycleBinExpanded">
```

Comment:
It is not a good practice to change the `@Input` property in the child component. I leave you to figure out the reason.

Reply:
Could the reason be that we want the child component to have a more predictable behaviour? Because if we let the child change the input property, the parent component will not have control over how the child will behave which is not ideal.

Anyway, have made the change to have the click emit an event which parent component can subscribe to instead.

Reply:
> Could the reason be that we want the child component to have a more predictable behaviour? Because if we let the child change the input property, the parent component will not have control over how the child will behave which is not ideal.

Exactly. However, this does not mean all logic should go to parent component. The child components can still make changes by using **event** (then the control becomes the parent component, either subscribe it or ignore it).

For example, in this case, if want to keep you toggle logic, you can first `isRecycleBinExpandedChange` event inside the child component.